webview/src/shared/components/webviewWrapper/styles.module.scss

```diff
@@ -1,4 +1,5 @@
 .webviewWrapper {
   width: 100%;
```

Comment:
Changing this to `min-content` would solve the following bug:


https://user-images.githubusercontent.com/3683420/201679946-a7cfc21a-6bc1-4be2-8c88-3dc6754e0eac.mov

Reply:
Technically, this bug is in main and not part of this pr. Could be fixed separately :)

Reply:
thanks @sroy3, good catch. The fix proposed changes the whole UX in a way that we probably don't want. Instead of the ribbon taking always size of the viewport only and overflowing items, it's now taking the whole space (defined by the most wide plot). I'm not sure this is a good idea. Also, plots size control will move to the edge if we do it this way

I can find a way to fill the space with something, but the whole layout might change after the #2747 and further changes?

Reply:
You are right, it is not as simple as that a change. What I'd do is make the ribbon `100%` and create a div inside (ribbon-content or something else) and give it a width of `100vw` (might have to calculate padding in there).

Reply:
let's skip it for now and get back to it after #2585 is done then